### PR TITLE
QA-171 Add ADCM URL attachment to the Allure Report if `--dontstop` is used

### PR DESCRIPTION
**Pull request by dgusakov**

### CODE REVIEW

File: src/adcm_pytest_plugin/fixtures.py [hunk 1]
```diff
@@ -208,6 +210,27 @@ def _attach_adcm_logs(request: SubRequest, adcm: ADCM):
             )
 
 
+def _attach_adcm_url(request: SubRequest, adcm: ADCM):
+    """Attach ADCM URL link to the Allure Report for the further access"""
+    attachment_name = "ADCM API URL"
```
Comment (vgorkavenko) [inline src/adcm_pytest_plugin/fixtures.py:215]: Most likely you meant the link to the admin panel and not API?

Comment (dgusakov) [reply src/adcm_pytest_plugin/fixtures.py:215]: Actually it is something like `http://{host}:{port}`

Comment (vgorkavenko) [reply src/adcm_pytest_plugin/fixtures.py:215]: I suggest leaving just `ADCM URL` without `API`

**Review by vgorkavenko — APPROVED** [review]

File: src/adcm_pytest_plugin/fixtures.py (hunk 1 above)
Comment (dgusakov) [reply src/adcm_pytest_plugin/fixtures.py]: Done

**Review by madlabman — APPROVED** [review]

**Pull request merged**